portmidi.go

```diff
@@ -88,5 +88,8 @@ func Time() Timestamp {
 
 // convertToError converts a portmidi error code to a Go error.
 func convertToError(code C.PmError) error {
```

Comment:
Shouldn't we handle this case at the call sites? Why do we call convertToError if code is non-negative?